dashboard/run_dashboard_app.py

```diff
@@ -126,6 +135,19 @@ def update_maui_total_chours(n):
     return fig
 
 
+@app.callback(Output('confirm', 'displayed'),
+              [Input("interval_comp", "n_intervals")])
+def display_confirm(n):
+    return app.db.get_collection_err(const.HPC.maui) is not None
+
+
+@app.callback(Output('output-confirm', 'children'),
+              [Input('confirm', 'submit_n_clicks')])
+def update_output(submit_n_clicks):
+    if submit_n_clicks:
+        return None
```

Comment:
Does this do anything? A function without a return statement returns a None value anyway.

Reply:
it returns None when you click the options in the alert box